x-pack/plugins/security_solution/public/resolver/view/panels/cube_for_process.tsx

```diff
@@ -28,7 +28,6 @@ export const CubeForProcess = memo(function CubeForProcess({
         height="1.5em"
         viewBox="0 0 1 1"
       >
-        <desc>{descriptionText}</desc>
```

Comment:
:red_circle: Since we're communicating meaning with the visual presentation of those cubes, I don't think it's right to remove the desc from that without at least trying to set up some alternative accessible description.

Reply:
i restored the `desc`